packages/swapper/src/swappers/thorchain/utils/txFeeHelpers/btcTxFees/getBtcTxFees.ts

```diff
@@ -1,49 +1,51 @@
-import { bitcoin, ChainAdapterManager } from '@shapeshiftoss/chain-adapters'
+import { ChainId } from '@shapeshiftoss/caip'
+import { ChainAdapter } from '@shapeshiftoss/chain-adapters'
 import { KnownChainIds } from '@shapeshiftoss/types'
 
 import { QuoteFeeData, SwapError, SwapErrorTypes } from '../../../../../api'
 import { bn } from '../../../../utils/bignumber'
+import { ThorChainUtxoChainIds } from '../../../types'
 
 export const getBtcTxFees = async ({
   opReturnData,
   vault,
   sellAmount,
-  adapterManager,
+  sellAdapter,
   pubkey,
   tradeFee
 }: {
   opReturnData: string
   vault: string
   sellAmount: string
-  adapterManager: ChainAdapterManager
+  sellAdapter: ChainAdapter<ChainId>
   pubkey: string
   tradeFee: string
 }): Promise<QuoteFeeData<KnownChainIds.BitcoinMainnet>> => {
   try {
-    const adapter = adapterManager.get('bip122:000000000019d6689c085ae165831e93') as
-      | bitcoin.ChainAdapter
-      | undefined
-    if (!adapter)
-      throw new SwapError(
-        `[getBtcTxFees] - No chain adapter found for bip122:000000000019d6689c085ae165831e93.`,
-        {
-          code: SwapErrorTypes.UNSUPPORTED_CHAIN,
-          details: { chainId: 'bip122:000000000019d6689c085ae165831e93' }
-        }
-      )
-    const feeDataOptions = await adapter.getFeeData({
+    const feeDataOptions = await (sellAdapter as ChainAdapter<ThorChainUtxoChainIds>).getFeeData({
       to: vault,
       value: sellAmount,
       chainSpecific: { pubkey, opReturnData }
     })
 
     const feeData = feeDataOptions['fast']
 
+    // BCH specific hack:
```

Comment:
I admit this is hacky. 

BUT I did not feel it was worth the time trying to figure out exactly whats going on (coinselect not working how we expect maybe?) when BCH is a less common coin AND fees are already very low for bch

Reply:
Yeah given the BCH fees I'm fine if this is unblocking us for now.

I'm wondering, is there a lower multiplier that we can use while still giving us confidence Txs will go through?

Can we create an issue to remove the hack in a follow-up refactor?

Reply:
We could probably lower the multiplier but I also dont care because 1) this works, 2) its a very low fee anyway (like 7 cents) and dont want to take the time to try and find the "perfect" multiplier that may be too low in some cases

I created an issue to track the hack but I also don't think fixing this needs to be a very high priority -- imo(we have more important things to worry about than sends from bch costing a few cents extra:

https://github.com/shapeshift/lib/issues/959